tests/src/test/scala/com/evolutiongaming/kafka/journal/IntegrationSuite.scala

```diff
@@ -13,6 +13,7 @@ import com.evolutiongaming.kafka.journal.IOSuite._
 import com.evolutiongaming.scassandra.CassandraClusterOf
 import com.evolutiongaming.smetrics.{CollectorRegistry, MeasureDuration}
 import com.typesafe.config.ConfigFactory
+import JsValueCodec.Implicits.default
```

Comment:
this is a nice way to test whether this all works as expected, so please run it at least locally with jsoniter :)